src/is-ipados.ts

```diff
@@ -0,0 +1,6 @@
+export = function isIpadOS(ua?: string, document?: object): boolean {
+  ua = ua || window.navigator.userAgent;
+  document = document || window.document;
+
+  return /Mac|iPad/i.test(ua) && "ontouchend" in document;
```

Comment:
do we think it's worth adding a comment, here, explaining what `ontouchend` is and why we use it to determine iPad vs non-iPad browsers?

Reply:
I sure can, I wasn't sure since the whole repo is a bit lighter on that

Reply:
Yeah, I think that's a reflection on past teams being much smaller and having more context. I'm ok with this getting documented in the code for future folks

Reply:
suggested changes for clarity:

  // "ontouchend" is used to determine if a browser is on an iPad, otherwise
  // user-agents for iPadOS behave/identify as a desktop browser

Reply:
done